packages/frontend/src/components/pages/search-page/searchPage_service.js

```diff
@@ -7,4 +7,13 @@ const getOrder = async (searchId) => {
   return response.data
 }
 
-export default getOrder
+const findOrder = async (searchData) => {
```

Comment:
функция возвращает массив заказов, поэтому лучше ее назвать findOrders
```suggestion
const findOrders = async (searchData) => {
```